cross/x86_ci_script.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+function exit_with_error {
+    set +x
+
+    local errorMessage="$1"
+    local printUsage=$2
+
+    echo "ERROR: $errorMessage"
+    exit 1
+}
+
+#Exit if input string is empty
+function exit_if_empty {
+    local inputString="$1"
+    local errorMessage="$2"
+    local printUsage=$3
+
+    if [ -z "$inputString" ]; then
+        exit_with_error "$errorMessage" $printUsage
+    fi
+}
+
+# Cross builds corefx using Docker image
+function cross_build_native_with_docker {
+    __currentWorkingDirectory=`pwd`
+
+    # choose Docker image
+    __dockerImage=" hseok82/dotnet-buildtools-prereqs:ubuntu1604_cross_prereqs_v3_x86"
```

Comment:
cc @gkhanna79 @danmosemsft @weshaggard 
Naively, central trusted location sounds like safer approach, but I don't have enough info about Core the infra.